### DISCUSSION TITLE
Unable to login and gets a 500 error

### DISCUSSION THREAD

**Discussion by win98se** [elk-zone/elk#3056] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/05409f7e-bc17-46ef-9d8b-114fbe6c57a0)

After login, I see this page directly and I can't proceed any further.

**Comment by johnsonm** [comment]
I'm seeing the same on main but not stable on Chrome on Android — #3072

**Reply by shuuji3** [reply, marked as answer]
This issue is now fixed: #3072 Thanks for your patience!
